new_client/src/components/Footer.css

```diff
@@ -1,46 +1,15 @@
 .footer {
-    float: left;
-    width: 100%;
-    background-color: #000000;
-    overflow: auto;
-    padding-left: 40px;
-  
+  display: flex;
+  justify-content: space-between;
+  padding: 10px 2%;
+  background-color: #0d0d0d;
+}
```

Comment:
As shown in the photos ,if we are keeping footer on all pages then we still need to align it.

Reply:
![image](https://user-images.githubusercontent.com/50794716/82153215-c61abc00-9883-11ea-888b-6cef01e864c8.png)

Reply:
![image](https://user-images.githubusercontent.com/50794716/82153264-ff532c00-9883-11ea-9171-65a72b601f63.png)

Reply:
Right. @Hrushi2406 please do this change